app/services/letter_sending_service.rb

```diff
@@ -48,18 +48,18 @@ def client
   end
 
   def notify_api_key
-    if Rails.configuration.production_environment
-      @notify_api_key ||= (@local_authority.notify_api_key || ENV.fetch("NOTIFY_API_KEY"))
-    else
+    if Bops.env.staging?
```

Comment:
In this one particular, I've changed the logic due to some specs failures.

Reply:
I'm pretty sure that `production_environment` was being set to `true` even when running in the development/test environment since this is how it was being set: 

``` ruby
config.production_environment = (ENV.fetch("STAGING_ENABLED", "false") == "false")
```

So unless `STAGING_ENABLED` was set to `true`, `production_environment` would be true.